### PR TITLE
[Web3 0.20] Add script to sweep balance to a new ETH wallet

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

File: bin/lamassu-eth-sweep-to-new-wallet [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  })
+  .then(([balance, settings, { account, operatorId }]) => {
+    const tx = {
+      cryptoCode: 'ETH',
+      toAddress: newAddress,
+      cryptoAtoms: BN(balance.times(0.99999).toFixed(0, BigNumber.ROUND_DOWN))
+    }
+
+    const opts = {
+      chainId: 3,
```
Comment (josepfo) [inline bin/lamassu-eth-sweep-to-new-wallet:266]: Should be 1 instead, right?

File: bin/lamassu-eth-sweep-to-new-wallet [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      console.log('Neither geth nor Infura are running, so the script cannot be executed.')
+      process.exit(2)
+    }
+
+    if (infuraIsRunning) {
+      const endpoint = _.startsWith('https://')(settings.accounts.infura.endpoint)
+        ? settings.accounts.infura.endpoint
+        : `https://${settings.accounts.infura.endpoint}`
+      connect(endpoint)
+    } else {
```
Comment (josepfo) [inline bin/lamassu-eth-sweep-to-new-wallet:252]: No need for else here

File: bin/lamassu-eth-sweep-to-new-wallet [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    console.log('New mnemonic stored successfully! All your funds (minus the transaction fee) should be available in the next few minutes.')
+    return fs.rmdir(LOCKFILE_PATH)
+  })
+  .then(() => {
+    console.log('lamassu-eth-pending-sweep lock file successfully removed, you may now execute the upgrade script again')
+    return fs.mkdir(`${LOCKFILE_PATH}-finished`)
+  })
+  .then(() => {
+    console.log('lamassu-eth-pending-sweep-finished lock file successfully created, this will automatically be deleted once the upgrade script finishes running')
+    console.log('Process finished successfully!')
```
Comment (josepfo) [inline bin/lamassu-eth-sweep-to-new-wallet]: Maybe the message `you may now execute the upgrade script again` would be better suited in this last log?

File: bin/lamassu-eth-sweep-to-new-wallet [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+      connect(`http://localhost:${coins.utils.getCryptoCurrency('ETH').defaultPort}`)
+    }
+
+    return Promise.all([getAllBalance(), settings, fetchWallet(settings, 'ETH')])
+  })
+  .then(([balance, settings, { account, operatorId }]) => {
+    const tx = {
+      cryptoCode: 'ETH',
+      toAddress: newAddress,
+      cryptoAtoms: BN(balance.times(0.99999).toFixed(0, BigNumber.ROUND_DOWN))
```
Comment (josepfo) [inline bin/lamassu-eth-sweep-to-new-wallet:262]: Can you explain this?

**Review by josepfo — APPROVED** [review]

File: bin/lamassu-eth-sweep-to-new-wallet [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+    return writeNewMnemonic(mnemonic)
+  })
+  .then(() => {
+    console.log('New mnemonic stored successfully! All your funds (minus the transaction fee) should be available in the next few minutes.')
+    return fs.rmdir(LOCKFILE_PATH)
+  })
+  .then(() => {
+    console.log('lamassu-eth-pending-sweep lock file successfully removed')
+    return fs.mkdir(`${LOCKFILE_PATH}-finished`)
+  })
```
Comment (RafaelTaranto) [inline bin/lamassu-eth-sweep-to-new-wallet]: We need to create and backup old mnemonics before moving funds.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Pull request merged**